### PR TITLE
Updates the dynamic ball handling challenge

### LINKED ISSUE #44: Updates on Dynamic ball handling challenge

### PR DESCRIPTION
**Pull request by PatrickGoettsch**
Closes #44

### CODE REVIEW

**Review by sseering — APPROVED** [review]

File: content/robot_players.tex [hunk 1]
```diff
@@ -17,7 +17,7 @@ \subsection{Hardware}
     \item A memory stick may remain in the head during operation.  Only ordinary USB flash memory keys that sit flush or recessed to the head casing may be utilized. Other USB dongles or devices, as well as memory sticks that are not flush or recessed, are not permitted.
 \end{itemize}
 
-A computer with two monitors (one for GC and one for TCM) will be provided by the event organizers for the purpose of sending GameController messages to the robots and observing if no robot violates the rules for Wifi usage.
+A computer with two monitors (one for GC and one for TCM) will be provided by the event organizers for the purpose of sending GameController messages to the robots and observing if no robot violates the rules for wireless network usage.
```
Comment (arnemoos) [inline content/robot_players.tex:20]: Should be not in this PR? Instead a commit on the rules themselves?

File: content/technical_challenges.tex [hunk 2]
```diff
@@ -29,8 +29,8 @@ \subsection{7 vs. 7}
             \end{figure}
             Blue and red jerseys with numbers from 1 to 7 are provided by the OC/TC. If a team decides to play with own jerseys, they have to provide an additional one with number 7 of the same shape. 
 
-        \paragraph{WiFi}
-            The available Wifi package budget defined in \cref{sec:wireless} will be increase by \qty{40}{\percent} both for the initial budget and for the budget for playing time extensions.  
+        \paragraph{Wireless network}
+            The available wireless network package budget defined in \cref{sec:wireless} will be increase by \qty{40}{\percent} both for the initial budget and for the budget for playing time extensions.
```
Comment (arnemoos) [inline content/technical_challenges.tex:33]: Should be not in this PR? Instead a commit on the rules themselves?

File: content/technical_challenges.tex [hunk 3]
```diff
@@ -90,10 +90,13 @@ \subsection{7 vs. 7}
...
-            \paragraph{After game} Teams have time of \qty{15}{\minute} to return the pool robots to the pool.
+            \begin{description}[leftmargin=0.5cm,labelindent=0.5cm]
+                \item[Robot check] Each team marks and checks its robots sent to the robot pool as described in \cref{sec:robot_pool_evaluation} and reports its availability (\qty{1.5}{\hour} before a match starts).
+                \item[Setup] One hour before the match starts, the teams receive their randomly selected robots. They have now time for set-up and calibration. For calibration only one half of the field is available for a team. The area within the center circle is not allowed to be entered by the robots until the match starts.
+
+                \item[Game] During a match additional rules apply for the pool robots. Referees are asked to try to save the robots' hardware by catching them before they crash onto the ground.
+                \item[After game] Teams have time of \qty{15}{\minute} to return the pool robots to the pool.
+            \end{description}
```
Comment (arnemoos) [inline content/technical_challenges.tex:99]: Should be not in this PR? Instead a commit on the rules themselves?

File: content/technical_challenges.tex [hunk 4]
```diff
@@ -237,11 +241,13 @@ \subsection{Dynamic Ball handling Challenge}
...
+
+        Since the challenge has three runs for each team, each team will be teamed with another team for a run. Following from this each run is divided into two phases. Executed after each other. For the first phase of a run the first mentioned team brings their three attacking robots and the other team provides the defending team. During the second phase teams switch their robots respectively.
 
-        Since the challenge has three runs for each team, each team will be teamed with another team for a run. Both teams have to provide the same three robots (except one robot breaks) as attacking as well as defending team. Following from this each run is divided into two parts. Executed after each other.
+        After the first and second run, attacking robots must be stored at the designated place switched off.
```
Comment (arnemoos) [inline content/technical_challenges.tex:248]: designated place **and** switched off

File: content/technical_challenges.tex [hunk 5]
```diff
@@ -258,45 +264,54 @@ \subsection{Dynamic Ball handling Challenge}
...
+            \item Ball leaves the field outside the defending goal area.
             \item An attacker pushes.
             \item A run exceeds \qty{4}{\minute} execution time.
-            \item A goal is scored.
+            \item A goal is scored after three robots have played the ball.
+            \item Attacking team is not communicating with the GameController.
         \end{itemize}
 
-        A score for the run will be calculated based on the following rules:
+        In the case that a goal gets scored before three robots played the ball, or the ball leaves the field within the defending goal area, the ball gets placed on the closest goal kick spot.
```
Comment (arnemoos) [inline content/technical_challenges.tex:299]: What is meant with "played" only touched or are the passes meant?

File: content/game_process.tex [hunk 6]
```diff
@@ -22,7 +22,7 @@ \subsection{Robot States}
 
 \textbf{The usage of the button interface as a replacement for any GameController commands/transitions is not allowed in the main competition!}
 
-Should, on both teams, at least two robots have problems with the Wifi/GameController connection the head referee should issue a referee timeout (see \cref{sec:referee_timeout}).
+Should, on both teams, at least two robots have problems with the wireless network or GameController connection the head referee should issue a referee timeout (see \cref{sec:referee_timeout}).
```
[the remark this answers is not in the document]
Comment (PatrickGoettsch) [reply content/game_process.tex:25]: Yeah, should have been, but I thought this PR has to be merged anyhow so I let it in.

File: content/game_process.tex [hunk 7]
```diff
@@ -394,7 +394,7 @@ \subsection{Penalty Kick Shoot-out}
...
+Before the penalty shootout begins, each team must hand over to referees up to 6 prepared robots that may participate in the penalty shootout. No robots may be added once the penalty shootout starts. Robots that will not participate in the shootout must not be on the wireless network and must stay outside of the field. All participating robots must be wearing the correct jersey for their player (1-6) and no duplicate numbers are permitted. Before each penalty kick, both teams must select the robot to participate (as goalkeeper or striker) in the penalty kick. The team leader communicates the selection to the head referee by privately handing the referee a card with their chosen number. After both teams have selected their player, the GameController operator selects the requested striker and goalkeeper robots from the opposing teams and the GameController communicates that all non-selected robots are substitutes and should remain inactive.
```
[the remark this answers is not in the document]
Comment (PatrickGoettsch) [reply content/game_process.tex:397]: Yeah, should have been, but I thought this PR has to be merged anyhow so I let it in.

File: content/technical_challenges.tex (hunk 2 above)
Comment (PatrickGoettsch) [reply content/technical_challenges.tex:33]: Yeah, should have been, but I thought this PR has to be merged anyhow so I let it in.

File: content/technical_challenges.tex [hunk 8]
```diff
@@ -203,7 +206,7 @@ \subsection{Visual Referee Challenge}
                 \includegraphics[height=120px]{figs/referee-signals/pushing-flipped.png}
             \end{subfigure}
             \caption{\textbf{Pushing Free-kick \textlangle{}color\textrangle{}  Team.}
-            Two-handed signal. One arm, vertical with bent elbow and palm facing in the direction of the extended arm. One arm, extended horizontally in the in the direction of the half of the field corresponding to the team that is \emph{given} the penalty. That is, right arm extended for the ``Blue team'', and left arm extended for the ``Red team''.}
+            Two-handed signal. One arm, vertical with bent elbow and palm facing in the direction of the extended arm. One arm, extended horizontally in the direction of the half of the field corresponding to the team that is \emph{given} the penalty. That is, right arm extended for the ``Blue team'', and left arm extended for the ``Red team''.}
```
[the remark this answers is not in the document]
Comment (PatrickGoettsch) [reply content/technical_challenges.tex:209]: Yeah, should have been, but I thought this PR has to be merged anyhow so I let it in.

File: content/robot_players.tex (hunk 1 above)
Comment (PatrickGoettsch) [reply content/robot_players.tex:20]: Yeah, should have been, but I thought this PR has to be merged anyhow so I let it in.

File: content/technical_challenges.tex (hunk 4 above)
Comment (PatrickGoettsch) [reply content/technical_challenges.tex]: Is now addressed

File: content/technical_challenges.tex (hunk 5 above)
Comment (PatrickGoettsch) [reply content/technical_challenges.tex]: A phrased it now differently

**Comment by PatrickGoettsch** [comment]
@arnemoos  and @sseering could you please have a look on the updates?

**Comment by sseering** [comment]
I agree with the intention of the changes.

I made some wording and grammar changes where I think it improved the language. I did my best to keep the meaning the same.

Feel free to change or revert my changes if you disagree with any.

**Review by sseering — APPROVED** [review]

**Comment by PatrickGoettsch** [comment]
LGTM. Thanks @sseering

**Pull request merged**